content-src/components/Spotlight/Spotlight.js

```diff
@@ -76,7 +93,7 @@ const Spotlight = React.createClass({
       blankSites.push(<li className="spotlight-item spotlight-placeholder" key={`blank-${i}`} />);
     }
     return (<section className="spotlight">
-      <h3 className="section-title">Featured</h3>
+      <h3 data-l10n-id="spotlight-title" className="section-title">Featured</h3>
```

Comment:
Here and in other places:  I didn't remove the original English strings from the JSX for now, but we should consider doing it.

For one, if kept around they tend to get out of sync pretty quickly.  Also, React renders a number of `<span>` elements around text content (especially when you interpolate expressions in `{…}`) to help with the reconciliation process.  These `<span>` elements are currently removed when L20n inserts the translation into the parent element, so they're don't serve their purpose at all and aren't needed.

There might be alternative approaches to integrating L20n with React which would operate on the virtual DOM instead of the actual DOM and thus preserve whatever React ends up rendering.